apps/react/src/store/genre/slice.ts

```diff
@@ -1,25 +1,59 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, Draft, PayloadAction, SerializedError } from '@reduxjs/toolkit';
+import { DEFAULT_ERROR_MESSAGE } from '@js-camp/react/config';
+import { CursorPagination } from '@js-camp/core/models/cursor-pagination';
+import { Genre } from '@js-camp/core/models/genre';
 
-import { fetchGenres } from './dispatchers';
-import { initialState } from './state';
+import {
+	fetchGenresListCursorWithParams,
+	fetchGenresListCursorOnly,
+	fetchGenresListCursorWithParamsTypePrefix,
+	fetchGenresListCursorOnlyTypePrefix,
+} from './dispatchers';
+import { genresAdapter, GenresStateType, initialState } from './state';
+
+/** * Handle pending. */
+function handlePending() {
+	return (state: Draft<GenresStateType>) => {
+		state.isLoading = true;
+	};
+}
+
+/** * Handle fulfilled. */
+export function handleFulfilled() {
```

Comment:
Looks like you don't need to export this function
```suggestion
function handleFulfilled() {
```